containers/runtime/php7/Dockerfile

```diff
@@ -30,12 +30,10 @@ RUN gpg2 --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A17031138
 RUN \curl -sSL https://get.rvm.io | bash -s stable
 
 # Install Ruby 2.5
-RUN apt-get update && apt-get install -y procps && apt-get clean
+RUN apt-get --allow-releaseinfo-change update && apt-get install -y procps && apt-get clean
 RUN /bin/bash -l -c "rvm install ruby-2.5"
 RUN /bin/bash -l -c "rvm use --default ruby-2.5"
 RUN /bin/bash -l -c "echo 'gem: --no-document' > ~/.gemrc"
-RUN /bin/bash -l -c "echo 'export PATH=/usr/local/rvm/bin:$PATH' >> ~/.bashrc"
-RUN /bin/bash -l -c "echo 'rvm --default use ruby-2.5' >> ~/.bashrc"
```

Comment:
On these two line, just want to make sure we still run the right version of ruby. :)

Reply:
Verified the ruby version is still 2.5 which is what we want.